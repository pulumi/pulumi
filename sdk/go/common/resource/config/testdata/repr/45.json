{"my:outer":{"inner":"0123456"}}