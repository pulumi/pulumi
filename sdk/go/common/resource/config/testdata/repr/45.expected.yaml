decrypt:
  my:outer: '{"inner":"0123456"}'
paths:
  my:outer:
    value:
      inner: "0123456"
    string: '{"inner":"0123456"}'
    redacted: '{"inner":"0123456"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAd/4IaAAEFaW5uZXIGc3RyaW5nDAkABzAxMjM0NTY=
    secure: false
    isObject: true
  my:outer.inner:
    value: "0123456"
    string: "0123456"
    redacted: "0123456"
    object: FBAABnN0cmluZwwJAAcwMTIzNDU2
    secure: false
    isObject: false
